packages/next/src/server/base-server.ts

```diff
@@ -519,7 +519,23 @@ export default abstract class Server<ServerOptions extends Options = Options> {
   ): Promise<void> {
     return getTracer().trace(
       BaseServerSpan.handleRequest,
-      async () => await this.handleRequestImpl(req, res, parsedUrl)
+      {
+        tracerName: [req.method, req.url].join(' '),
+        kind: SpanKind.SERVER,
+        attributes: {
+          'http.method': req.method,
+          'http.target': req.url,
```

Comment:
[Is there any way we could set `http.route` here](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/http.md#http-server-semantic-conventions)?

Without this it's impossible to do aggregate analysis on paramaterized routes, since they would all have different URLs, which is something APM providers (like Sentry for example) would need!

Reply:
Hi, having `http.route` in this span is tricky.

Is adding it to a a nested span enough? Or do you need it at the `SERVER` span?

Reply:
Ideally everywhere possible. This is one of the big advantages of Sentry's custom perf implementation, and it would be hard for us to drop it for otel only if this wasn't the case.

We actually ended up injecting in paramaterized routes (`/pages/[id]` for ex.) via a webpack loader since next.js doesn't expose this very well: https://github.com/getsentry/sentry-javascript/blob/8343eb0d13fa83cbb850d4d026d5ace3a26d1201/packages/nextjs/src/config/loaders/wrappingLoader.ts#L97

Reply:
Ok, thanks for the feedback. Let's solve this in a follow-up PR.

Reply:
https://github.com/vercel/next.js/issues/47113